charon/src/ast/meta.rs

```diff
@@ -140,12 +165,17 @@ pub struct ItemMeta {
     /// API (this is called "pub-in-priv" items). With or without the `pub use`, we set `public =
     /// true`; computing item reachability is harder.
     pub public: bool,
-    /// Attribute `charon::rename("...")` (and `aeneas::rename("...")`) to provide a custom
-    /// name that can be used for instance when extracting the code via Aeneas.
-    pub rename: Option<String>,
-    #[serde(skip_serializing)]
-    #[serde(default)]
-    pub opaque: bool,
+}
+
+/// Meta information about an item (function, trait decl, trait impl, type decl, global).
+#[derive(Debug, Clone, Serialize, Deserialize, Drive, DriveMut)]
+pub struct ItemMeta {
+    pub span: Span,
+    pub name: Name,
```

Comment:
Do you really want to put this here? I understand that we may want to factor the name out. But is it really "meta" information?

Reply:
:shrug: clonse enough for me, I don't think this introduces ambiguity. The name feels definitely more meta than the signature or the body.